.github/workflows/rust_lint.yml

```diff
@@ -20,18 +20,22 @@ jobs:
       - uses: actions/checkout@v3
         with:
           submodules: recursive
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      # NOTE: This is needed to ensure that hyperdrive-wrappers builds correctly.
+      - name: install foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
 
       - name: check out hyperdrive
-        uses: actions/checkout@main
+        uses: actions/checkout@v3
         with:
           repository: delvtech/hyperdrive
-          ref: "latest"
```

Comment:
What happens if there's no ref? Should we be targeting a specific version?

Reply:
good catch; I pushed up a fix for this to grab use the latest tag.